### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Settings_Application_UseofColor: "Application" Link visually looks like text. There's no difference between link and other text content on page. 

### ISSUE THREAD

**Issue by Jtith** [microsoft/AzureStorageExplorer#6374]
**GitHub Tags:**

#A11yMAS;#A11yTCS;#MAC;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_Mac_Dec2022;#Azure Tools Storage Explorer;#WCAG1.4.1;#FTP;#AINotInScope;
## Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.27.0
## OS Details
Mac version: macOS Ventura 13.1

## Repro Steps:

1.  Launch Storage Explorer
2. Tab to Setting menu on left pane and click on it.
3. Tab to Application on right pane and  click on it.
4. "Application" screen gets opened.
5. Turn on voiceover and observe application, data explorer and other content on page are given as link but visually looks like a text. 

## Actual results:
 "Application" Link visually looks like text. There's no difference between link and other text content on page. 

## Expected results:
 "Application" Link should look like link. There should be a difference between link and other text content on page. 

##User Impact:
Low vision user will face difficulty in understanding content present on the screen. 
## Attachments:
<img width="2560" alt="MicrosoftTeams-image (31)" src="https://user-images.githubusercontent.com/100552367/207862927-bfb01af0-edcb-4439-9af6-216d4c4b1f20.png">

**Comment by craxal** [comment]
It would be more appropriate for the "links" here to be buttons. I will add some button styling for hover and focus so that it's more apparent that they are interactive.

**Comment by Jtith** [comment]
verified in the latest environment. Issue is fixed, Hence closing the bug
<img width="1440" alt="MicrosoftTeams-image (6)" src="https://user-images.githubusercontent.com/100552367/211790967-18fed353-57ad-4d12-a590-a8370fbccc4d.png">

**Comment by fsteffi** [comment]
GitHubTags:#Closed;
